file-formats/enho/README.md

```diff
@@ -1,9 +1,6 @@
-# File Format for Object Type ENHO
+# ENHO File Format
 
-For ENHO (BAdI Enhancement Implementation), there is a metadata file.
-The JSON schema for the ENHO metadata file is provided [here](./enho.json).
 
-Example files for an ENHO `z_aff_example_enho` are provided in the following table.
-File | Content
- --- | ---
-[`z_aff_example_enho.enhs.json`](./examples/z_aff_example_enho.enho.badi.json)    | Metadata of the ENHO
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enho.json`](./examples/z_aff_example_enho.enho.badi.json)
```

Comment:
```suggestion
`<name>.enho.json` | 1 | [`zif_aff_enho_v1.intf.abap`](./type/zif_aff_enho_v1.intf.abap) | [`enho.json`](./enho.json) | [`z_aff_example_enho.enho.badi.json`](./examples/z_aff_example_enho.enho.badi.json)
```

Reply:
But what about the `<name>.enho.json`. Is it still valid?

Reply:
Your are right. According to the example, this has to be `<name>.enho.badi.json`